src/package.lisp

```diff
@@ -166,15 +166,15 @@ multiplot etc."
           (t
            (format *user-stream* "~A " (gp-quote arg))))))
 
-(defmacro with-plots ((stream &key debug (external-format :default))
+(defmacro with-plots ((stream &key debug (external-format :default) (persist nil))
```

Comment:
maybe it is time to add a docstring to with-plots. According to the manual `-persist` is for keeping the GUI window alive after the command finishes

Reply:
Will add a doc string along with typo fix and COND logic as per above comment.